src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -2517,5 +2520,52 @@ public void Bug9288_keycontainer()
         // get to the third case (but there's still error handling if that changes)
 
         #endregion
+
+        #region PathMap Linux Tests
+        // Like the above (CoreCLR Signing Tests), these aren't actually syntax tests, but this is in one of only two assemblies tested on linux
+
+        [Fact]
```

Comment:
Did you consider using an xunit theory here vs. a fact? Seems like this is a fairly data driven test and would be ripe for a theory. 

Note: we don't have a policy here and I'm fine as is. But Theory seems ot be a feature other repos are more apt to use than Roslyn is.

Reply:
No, I didn't consider it (forgot that it existed). I was about to say "nah, there's some wacky behavior here, so 4 parameters would be needed `(expected key, expected value, input key, input value)`", but then I tried it and it's not too bad to have 4 params.